OpenEdXMobile/res/layout/row_discussion_comments_comment.xml

```diff
@@ -28,13 +27,19 @@
             style="@style/discussion_regular_text"
             tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
 
-        <TextView
-            android:id="@+id/discussion_comment_count_report_text_view"
-            style="@style/discussion_responses_small_text"
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_gravity="end"
-            android:background="?android:selectableItemBackground"
-            android:clickable="true"
-            android:padding="@dimen/discussion_responses_box_padding"
-            tools:text="@string/discussion_responses_report_label" />
+            android:minHeight="@dimen/discussion_responses_box_padding">
```

Comment:
How about adding a comment on top of this container, explaining the purpose of it?